src/librustc_passes/check_const.rs

```diff
@@ -0,0 +1,136 @@
+//! This pass checks the HIR bodies of const items (e.g., `const`, `static`, `const fn`) for
```

Comment:
```suggestion
//! This pass checks the HIR bodies of const contexts (e.g., `const`, `static`, `const fn`) for
```

Reply:
I reverted this change in the last commit after rereading the module docs this morning. HIR bodies are tied to an item in my mental model.

Reply:
Ah; I'm interpreting "item" in the sense of the syntactic notion accepted by the macro fragment `:item` whereas for example anonymous constants are not items but they are const contexts.

Reply:
"HIR bodies that must be executable at compile time"?

There's a struct called `hir::Item` that has a `hir::Body` which is tripping me up I think.

Reply:
Sure, a tad long, but it works. :)